Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMeraki.yaml

```diff
@@ -1,46 +1,580 @@
+        "73", "CPHB", "",
+        "74", "WSN", "",
+        "75", "PVP", "",
+        "76", "BR-SAT-MON", "",
+        "77", "SUN-ND", "",
+        "78", "WB-MON", "",
+        "79", "WB-EXPAK", "",
+        "80", "ISO-IP", "",
+        "81", "VMTP", "",
+        "82", "SECURE-VMTP", "",
+        "83", "VINES", "",
+        "84", "IPTM", "",
+        "85", "NSFNET-IGP", "",
+        "86", "DGP", "",
+        "87", "TCF", "",
+        "88", "EIGRP", "",
+        "89", "OSPFIGP", "",
+        "90", "Sprite-RPC", "",
+        "91", "LARP", "",
+        "92", "MTP", "",
+        "93", "AX.25", "",
+        "94", "IPIP", "",
+        "95", "MICP", "",
+        "96", "SCC-SP", "",
+        "97", "ETHERIP", "",
+        "98", "ENCAP", "",
+        "99", "", "",
+        "100", "GMTP", "",
+        "101", "IFMP", "",
+        "102", "PNNI", "",
+        "103", "PIM", "",
+        "104", "ARIS", "",
+        "105", "SCPS", "",
+        "106", "QNX", "",
+        "107", "A/N", "",
+        "108", "IPComp", "",
+        "109", "SNP", "",
+        "110", "Compaq-Peer", "",
+        "111", "IPX-in-IP", "",
+        "112", "VRRP", "",
+        "113", "PGM", "",
+        "114", "", "",
+        "115", "L2TP", "",
+        "116", "DDX", "",
+        "117", "IATP", "",
+        "118", "STP", "",
+        "119", "SRP", "",
+        "120", "UTI", "",
+        "121", "SMP", "",
+        "122", "SM", "",
+        "123", "PTP", "",
+        "124", "ISIS over IPv4", "",
+        "125", "FIRE", "",
+        "126", "CRTP", "",
+        "127", "CRUDP", "",
+        "128", "SSCOPMCE", "",
+        "129", "IPLT", "",
+        "130", "SPS", "",
+        "131", "PIPE", "",
+        "132", "SCTP", "",
+        "133", "FC", "",
+        "134", "RSVP-E2E-IGNORE", "",
+        "135", "Mobility Header", "",
+        "136", "UDPLite", "",
+        "137", "MPLS-in-IP", "",
+        "138", "manet", "",
+        "139", "HIP", "",
+        "140", "Shim6", "",
+        "141", "WESP", "",
+        "142", "ROHC", "",
+        "143", "Ethernet", "",
+        "144", "AGGFRAG", "",
+        "145", "NSH", "",
+        "253", "", "",
+        "254", "", "",
+        "255", "Reserved", ""
+    ];
+    let seveirtylookupforpriority=datatable(priority: string, EventSeverity: string)[
+        "1", "High",
+        "2", "Medium",
+        "3", "Low",
+        "4", "Informational"
+    ];
+    let severitylookupforDvcAction=datatable(DvcAction: string, EventSeverity: string)[
+        "Allow", "Informational",
+        "Deny", "Low"
+    ];
+    let directionlookup=datatable(direction: string, NetworkDirection: string)[
+        "ingress", "Inbound",
+        "egress", "Outbound",
+        "Unknown", "Unknown"
+    ];
+    let parser=(disabled: bool=false) {
+        union isfuzzy=true
+            (
+            meraki_CL
+            | project-rename LogMessage =  Message
+            ),
+            (
+            Syslog
+            | project-rename LogMessage =  SyslogMessage
+            )
```

Comment:
As discussed, watchlist to filter source is required here.

Reply:
Done